### ISSUE TITLE
Release 0.1.0 to METADATA.jl

### ISSUE THREAD

**Issue by paulhendricks** [JuliaML/MLMetrics.jl#15]

**Comment by paulhendricks** [comment]
Will release once this issue is resolved: https://github.com/JuliaLang/METADATA.jl/issues/4951

**Comment by Evizero** [comment]
any progress?

**Comment by Evizero** [comment]
bump. any interest in continuing this? Otherwise I can look into this

**Comment by Evizero** [comment]
@paulhendricks Unless you object I will take care of this for now. I am assuming you are busy with other projects or new adventures. Either way, thanks for your work so far

**Comment by paulhendricks** [comment]
@Evizero 

I think that's for the best. Life has thrown me a couple curve balls and I unfortunately don't have the time to devote to this. You have my full permission to do with this project whatever you feel is within the best interest of JuliaML.

Cheers.
Paul

**Comment by Evizero** [comment]
Yes, such is life! We will try to take good care of this for the time being. You know where to find us. good luck with whatever you are up to currently!
